Terasology/LightAndShadow
# Gameover screen

Adds an Overlay Screen showing whether you lost or won a round.

# How to test:
1. Start a game with LaS gameplay mode.
2. Choose a faction.
4. To see 'lost' screen too, you can join a second player too in the opposing faction.
3. Retrieve the flag 2 times to win and view the game over screen. If you had a second player, it will also have a game over screen with 'You lost' written.
